apisix/plugins/openid-connect.lua

```diff
@@ -152,26 +225,28 @@ function _M.access(plugin_conf, ctx)
             core.log.error("failed to introspect in openidc: ", err)
             return response
         end
-        if response then
```

Comment:
Why remove this part?

Reply:
The reason is that adding the user info header is now handled in the `introspect` method. The header will still be added, if so configured, and introspection is done via the actual endpoint as opposed to just checking against the public key. Let me know if you see any problem with this approach.